docs/content/concepts/automation.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: "Automation | Dagster Docs"
+description: "Learn to automatically run your Dagster pipelines."
+---
+
+# Automation
+
+Dagster offers several ways to run data pipelines without manual intervation, including traditional scheduling and event-based triggers. Automating your Dagster pipelines can boost efficiency and ensure that data is produced consistently and reliably.
+
+When one of Dagster's automation methods is triggered, a **tick** is created, which indicates that a **run** should occur. The tick will kick off a run, which is a single instance of a pipeline being executed.
+
+In this guide, we'll cover the available automation methods Dagser provides and when to use each one.
+
+---
+
+## Prerequisites
+
+Before continuing, you should be familiar with:
+
+- [Software-defined Assets][assets]
+- [Jobs][jobs] (_optional_)
+- [Ops][ops] (_optional; advanced_)
+
+---
+
+## Available methods
```

Comment:
I think it might help to talk a little about declarative vs imperative approaches to scheduling. Something like:

"Dagster offers several ways to automate your data pipelines. 

Schedules is our imperative approach, which allows you specify exactly when you would like a particular set of assets defined by a job to run. 

Sensors is an event-based approach that is well-suited to changes to an outside environment ....

AMP is our declarative system, where you assign policies to assets and allow Dagster to determine the best approach to keeping your assets up-to-date to adhere to those policies. You can apply both time-based policies, as well as more complex policies that .... etc
"